webapp-django/crashstats/crashstats/views.py

```diff
@@ -782,11 +778,12 @@ def crashes_per_day(request, default_context=None):
     # to be called `version`
     supersearch_params = copy.deepcopy(params)
     supersearch_params['version'] = supersearch_params.pop('versions')
-    supersearch_params['platform'] = supersearch_params.pop('platforms')
-    # in SuperSearch it's called 'Mac' not 'Mac OS X'
-    if 'Mac OS X' in supersearch_params['platform']:
-        supersearch_params['platform'].append('Mac')
-        supersearch_params['platform'].remove('Mac OS X')
+    if 'platforms' in supersearch_params:
+        supersearch_params['platform'] = supersearch_params.pop('platforms')
+        # in SuperSearch it's called 'Mac' not 'Mac OS X'
+        if 'Mac OS X' in supersearch_params['platform']:
+            supersearch_params['platform'].append('Mac')
+            supersearch_params['platform'].remove('Mac OS X')
```

Comment:
I prefer your hackish option since that's clear in its intentions and has a comment.